hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/executorservice/ExecutorServiceCancelOnPartitionMessageTask.java

```diff
@@ -35,19 +31,15 @@ public ExecutorServiceCancelOnPartitionMessageTask(ClientMessage clientMessage,
     }
 
     @Override
-    protected InvocationBuilder createInvocationBuilder() throws UnknownHostException {
-        final InternalOperationService operationService = nodeEngine.getOperationService();
-        final String serviceName = DistributedExecutorService.SERVICE_NAME;
-        CancellationOperation op = new CancellationOperation(parameters.uuid, parameters.interrupt);
-        return operationService.createInvocationBuilder(serviceName, op, parameters.partitionId);
+    protected Operation prepareOperation() {
+        return new CancellationOperation(parameters.uuid, parameters.interrupt);
```

Comment:
What about `parameters.partitionId`? I don't see it's used anywhere.

Reply:
This is really not needed, since the `AbstractMessageTask` retrieves the partition id from the clientMessage in `getPartitionId()` method. The client sets this partition id when sending the message. This parameter is duplication unless you set a different partition id on the message.